### PR TITLE
Add support for AKCP SecurityProbe

### PR DESCRIPTION
**Pull request by olivierbeytrison**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by murrant** [comment]
You need to add another snmprec file `tests/snmpsim/akcp-securityprobe.snmprec` with sysObjectID and sysDescr in it and update `tests/OSDiscovery.php`.  Otherwise, we could break this detection on accident.

**Comment by olivierbeytrison** [comment]
okay, i'll do it

File: tests/snmpsim/akcp-securityprobe.snmprec [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|securityProbe
```
Comment (laf) [inline tests/snmpsim/akcp-securityprobe.snmprec]: This needs to be the full sysDescr. I can see some examples in our stats:

securityProbe SEC-MX25V405n Jun 02 2016 10:57:49
securityProbe 5E SEC-PXAV405j Jun 25 2015 12:45:48

Comment (olivierbeytrison) [inline tests/snmpsim/akcp-securityprobe.snmprec]: hmm sure, it's a simulator =P
updating the commits

**Pull request merged**